app/javascript/components/explore/StudyGeneField.js

```diff
@@ -80,48 +77,60 @@ export default function StudyGeneField({ genes, searchGenes, allGenes }) {
     }
   }
 
+  /** handles a user selecting a gene list file to use */
+  function readGeneListFile(file) {
+    const fileReader = new FileReader()
+    fileReader.onloadend = () => {
+      const newGenes = fileReader.result.trim().split(/[\s,]+/)
```

Comment:
I just tested, so I know it works without it, but I'm surprised we don't need to call `split()` with `\n` here.

Reply:
`\s` gets whitespace characters, so it includes  `\n`